docs/core/releases-and-support.md

```diff
@@ -6,6 +6,9 @@ ms.topic: overview
 ---
 # Releases and support for .NET
 
+ > [!NOTE]
+ > This topic concerns .NET 5 and later versions, and versions of .NET Core, which preceded them. For .NET Framework releases, see [.NET Framework Lifecycle](https://learn.microsoft.com/en-us/lifecycle/products/microsoft-net-framework).
```

Comment:
```suggestion
 > This article concerns .NET 5 and later versions, and versions of .NET Core. For .NET Framework releases, see [.NET Framework Lifecycle](https://learn.microsoft.com/en-us/lifecycle/products/microsoft-net-framework).
```

Reply:
I added something similar in https://github.com/dotnet/docs/pull/37459/files#diff-ca0bc46f959dcebe0ffc031205a1cf11dc60771902da75a4375f0d24e9317323, so I think this is great.